app/scripts/controllers/overview.js

```diff
@@ -1350,11 +1352,11 @@ function OverviewController($scope,
     }, {poll: true, pollInterval: DEFAULT_POLL_INTERVAL}));
 
     if ($scope.AEROGEAR_MOBILE_ENABLED) {
-      watches.push(DataService.watch({ group: "mobile.k8s.io", version: "v1alpha1", resource: "mobileclients" }, context, function (clients) {
+      watches.push(MobileClientsService.createWatcher(context, function(clients) {
         overview.mobileClients = clients.by("metadata.name");
         updateFilter();
         Logger.log("mobileclients (subscribe)", clients);
-      }, { poll: limitWatches, pollInterval: DEFAULT_POLL_INTERVAL }));
+      }, {poll: limitWatches, pollInterval: DEFAULT_POLL_INTERVAL}));
```

Comment:
Just a question about why we need to have this polling?

Reply:
Im guessing it is there in case someone is using IE/Edge that doesnt support websockets.
I left it in as it was in @maleck13 changes, Craig do we keep those opts or are they safe to scrap?